include/webserv.hpp

```diff
@@ -57,4 +57,6 @@
 
 // Functions
 
+extern volatile bool g_sigint;
```

Comment:
You use it to check the status of a signal using a global variable, right? Good!